core/shared/src/main/scala/matryoshka/package.scala

```diff
@@ -35,101 +35,103 @@ import scalaz._, Scalaz._
   * @groupdesc dist Natural transformations required for generalized folds and unfolds.
   */
 package object matryoshka {
+  type Fix[F[_]] = matryoshka.data.Fix[F]
+  val Fix        = matryoshka.data.Fix
```

Comment:
Ok. I didn't actually mean to leave that alias in there, I'll pull it out. But it is a gargantuan task to adapt quasar, partly because there's no single place to drop a temporary in - many files don't import quasar.Predef - so you have to fix a million files at once.